packages/wallet-adapter-core/src/LegacyWalletPlugins/types.ts

```diff
@@ -74,7 +79,7 @@ export declare interface WalletCoreEvents {
   readyStateChange(wallet: Wallet): void;
   standardWalletsAdded(wallets: Wallet | AptosStandardSupportedWallet): void;
   networkChange(network: NetworkInfo | null): void;
-  accountChange(account: AccountInfo | null): void;
+  accountChange(account: AccountInfo | StandardAccountInfoInput | null): void;
```

Comment:
This is tricky as the adapter still supports legacy and new standard types, so the account the adapter gets back from the wallet can be of both types. Changing it gets us into a type-rabbit-hole, so we simply change the actual input type the adapter returns.

In the near future, we should probably support _only_ AIP-62 wallets

Reply:
I think we should use `AccountInfo` from `@aptos-labs/wallet-standard` here, that uses the `PublicKey` type.

If I understand correctly, this is an event that can be listened to externally by the dapp.
Since we're bumping the major, we might as well clean up the interface and have the "right" type here.

In case of legacy plugins, we should be able to adapt the return types to `AccountInfo` so that the dapp sees one consistent return type.